src/ia-bookmark-edit.js

```diff
@@ -52,12 +79,23 @@ export class IABookmarkEdit extends LitElement {
   }
 
   changeColorTo(id) {
-    this.bookmark.color = id;
+    this.newColor = id;
+    this.disableSave = this.shouldDisableSave();
     this.emitColorChangedEvent(id);
   }
 
   updateNote(e) {
-    this.bookmark.note = e.currentTarget.value;
+    this.newNote = e.currentTarget.value;
+    this.disableSave = this.shouldDisableSave();
+  }
+
+  shouldDisableSave() {
+    const colorHasChanged = this.colorStart !== this.newColor;
+    const noteHasChanged = this.noteStart !== this.newNote;
+    if (!colorHasChanged && !noteHasChanged) {
+      return true;
+    }
+    return false;
```

Comment:
Minor, but this can be

```suggestion
    return !colorHasChanged && !noteHasChanged; 
```